### PR TITLE
used for code review process.

### PR DESCRIPTION
**Pull request by DennisBerntsson**

### CODE REVIEW

File: dummy-file.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+```Python
+def test_func():
```
Comment (aannalarsson) [inline dummy-file.md:2]: This function needs a docstring.

Comment (DennisBerntsson) [reply dummy-file.md:2]: fixed the docstring

Comment (aannalarsson) [reply dummy-file.md:2]: ok!

File: dummy-file.md [hunk 2]
```diff
@@ -1,4 +1,13 @@
 ```Python
 def test_func():
+"""
+Here is a requested docstring.
+"""
 	print("Is this a good function?")
+	
+def is_this_a_book(image):
```
Comment (aannalarsson) [inline dummy-file.md:8]: Docstring?

File: dummy-file.md [hunk 3]
```diff
@@ -1,4 +1,13 @@
...
+	if(image == book):
```
Comment (aannalarsson) [inline dummy-file.md:9]: Need a comment

Comment (DennisBerntsson) [reply dummy-file.md:9]: No it does not. It is self explanatory.

Comment (aannalarsson) [reply dummy-file.md:12]: I want a comment

Comment (DennisBerntsson) [reply dummy-file.md]: added comment

File: dummy-file.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+"""
+Here is a requested docstring.
+"""
+	print("Is this a good function?")
+	
+def is_this_a_book(image):
+"""
+Here is a requested docstring.
+"""
+	if(image == book):  # This statement checks if a image is a book (this is a stupid comment...)
```
Comment (aannalarsson) [inline dummy-file.md:12]: Very good comment (Not stupid)

File: dummy-file.md (hunk 3 above)
Comment (DennisBerntsson) [reply dummy-file.md]: outdated

File: dummy-file.md (hunk 4 above)
Comment (DennisBerntsson) [reply dummy-file.md:12]: kewl

**Review by aannalarsson — APPROVED** [review]

**Pull request closed without merging**